src/Authentication/AzureArtifacts.cs

```diff
@@ -53,6 +53,11 @@ public static PublicClientApplicationBuilder WithBroker(this PublicClientApplica
                 })
             .WithParentActivityOrWindow(() => GetConsoleOrTerminalWindow());
```

Comment:
does this need to be something like:
```
.WithParentActivityOrWindow(() =>  parentWindowHandle ?? GetConsoleOrTerminalWindow());
```
?

Reply:
You're right. Seems like I've missed that stuff during conflict resolution